index.js

```diff
@@ -405,6 +408,15 @@ function normalizeOptions (opts) {
   // process
   normalized.serverUrl = parseUrl(normalized.serverUrl)
 
+  if (containerInfo) {
+    if (!normalized.containerId && containerInfo.containerId) {
+      normalized.containerId = containerInfo.containerId
+    }
+    if (!normalized.kubernetesPodUID && containerInfo.podId) {
+      normalized.kubernetesPodUID = containerInfo.podId
+    }
```

Comment:
Agreed on call: Add if block checking `!normalized.kubernetesPodName && containerInfo.podId` and if so set the pod name to the hostname